app/models/itunes_image.rb

```diff
@@ -1,5 +1,6 @@
 class ITunesImage < ActiveRecord::Base
   belongs_to :podcast
+  attr_accessor :link, :description
```

Comment:
so the single image representer can handle both, in another PR I actually unify this under one table with all the same columns, this is a stop gap